src/overlays/actors/ovl_En_Kakasi/z_en_kakasi.c

```diff
@@ -188,8 +188,10 @@ void EnKakasi_Init(Actor* thisx, PlayState* play) {
     i = 0;
     csId = this->picto.actor.csId;
     while (csId != CS_ID_NONE) {
-        //! FAKE:
-        csId = CutsceneManager_GetAdditionalCsId(this->csIdList[i] = csId);
+        // clang-format off
+        this->csIdList[i] = csId; \
```

Comment:
Perhaps still a fake, but I prefer this over what it was. Possibly a macro of some kind?